File: chatbot_application/dependencies/lang_chain.py
```python
from langchain import LangChain

class LangChainIntegration:
    def __init__(self, model_name):
        self.model_name = model_name
        self.lang_chain = LangChain(model_name)

    def generate_response(self, prompt):
        response = self.lang_chain.generate(prompt)
        return response

    def update_model(self, new_model_name):
        self.model_name = new_model_name
        self.lang_chain = LangChain(new_model_name)
```

File: backend-api/utils/langchain.js
```javascript
const LangChain = require('langchain');

const langChain = new LangChain();

const generateResponse = async (message) => {
  try {
    const response = await langChain.generateResponse(message);
    return response;
  } catch (error) {
    console.error(error);
    return null;
  }
};

module.exports = {
  generateResponse,
};
```